src/private/CABasicAnimation+MotionAnimator.m

```diff
@@ -211,7 +222,9 @@ void MDMConfigureAnimation(CABasicAnimation *animation, MDMMotionTiming timing)
       CGFloat displacement = -biggestDelta;
       CGFloat absoluteInitialVelocity =
           timing.curve.data[MDMSpringMotionCurveDataIndexInitialVelocity];
-      springAnimation.initialVelocity = absoluteInitialVelocity / displacement;
+      if (fabs(displacement) > CGFLOAT_EPSILON) {
```

Comment:
Please consider just checking that it's greater than some sensible constant, like 0.00001.  If I understand correctly, FLT_EPSILON is not intended to be used in this manner (it's not the smallest non-zero positive value).  I think using a 0.000001 or similar would be more straightforward to maintainers who do not need to go and investigate the complexities of floating point epsilon values.

https://stackoverflow.com/questions/10334688/how-dangerous-is-it-to-compare-floating-point-values

Reply:
Another good read, forwarded by @willlarche 

https://www.jessesquires.com/blog/floating-point-swift-ulp-and-epsilon/

Reply:
Done.